src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/RawEventTransformer.scala

```diff
@@ -31,15 +31,15 @@ import Scalaz._
 /**
  * Thrift serializer/deserializer class
  */
-class RawEventTransformer extends ITransformer[ ValidatedRecord, EmitterInput ] {
+class RawEventTransformer extends ITransformer[ValidatedRecord, EmitterInput] {
 
-  val log = LogFactory.getLog(getClass)
+  val log = LoggerFactory.getLogger(getClass)
 
   lazy val serializer = new TSerializer()
   lazy val deserializer = new TDeserializer()
 
   override def toClass(record: Record): ValidatedRecord = {
-    log.info("Converting one record to EmitterInput before adding it to the buffer")
+    log.debug("Converting one record to EmitterInput before adding it to the buffer")
```

Comment:
This logging line is a bit useless in its current form as all it states is that a record arrived.  It would be more useful to actually log the record as well in this case so someone trying to figure out what is occurring has access to the raw record as well.